support-frontend/assets/pages/[countryGroupId]/guardianAdLiteLanding/components/guardianAdLiteCards.tsx

```diff
@@ -40,13 +40,7 @@ export function GuardianLightCards({
 			aria-labelledby={`monthly`}
 		>
 			{cardsContent.map((cardContent, cardIndex) => {
-				return (
-					<GuardianLightCard
-						cardIndex={cardIndex}
-						key={`guardianLightCard${cardIndex}`}
-						{...cardContent}
-					/>
-				);
+				return <GuardianAdLiteCard cardIndex={cardIndex} {...cardContent} />;
```

Comment:
Where did the key go?

Reply:
I removed it as it was'nt referenced in the component when I checked. Is that ok to do?

Reply:
React lists usually need a unique key
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Reply:
Ah, understand, thanks. Key is auto-generated usually but a better naming conevntion used with the Cards. Have put it back with Guardaina Ad-Lite. Corrected